app/konnect/servicehub/manage-services.md

```diff
@@ -61,6 +61,22 @@ box to save.
 
 4. Enter a version name and click **Create** to save.
 
+### Archive a Service Version
+
+1. From the left navigation menu, click **Services**.
+
+2. Select a Service from the dropdown menu.
+
+3. Navigate to **Versions**.
+
+4. Click on the version you want to archive, and you'll be taken to the version detail page.
+
+5. Click on the **Actions** dropdown menu.
+
+6. Click **Archive Version** to archive the Service version, thereby preventing that version from showing up on the UI. Users will not be able to send requests to the Service version endpoints and those endpoints will no longer be supported.
```

Comment:
We could be more specific that it will prevent that version from showing up on the `Dev Portal UI`, but still show in the `Konnect Admin UI`. But this is also implied later with "An Archived version can be changed back to Published status", so just a thought.

Reply:
Let's be explicit and specify `Dev Portal UI` and that it's still visible in the Admin UI.

Reply:
This has been updated.